.gitignore

```diff
@@ -4,7 +4,6 @@
 # Ignore some folders
 node_modules
 /static/framework
-dist
```

Comment:
The discussion is open. I'd rather have `dist` ignored in the root `.gitignore` than having a `.gitignore` per component. When we'll publish the components to npm, then we may reconsider it.